src/helm/proxy/services/remote_service.py

```diff
@@ -41,6 +42,10 @@ def get_general_info(self) -> GeneralInfo:
         response = requests.get(f"{self.base_url}/api/general_info").json()
         return from_dict(GeneralInfo, response)
 
+    def get_tokenizer_info(self, model_name) -> TokenizerInfo:
```

Comment:
Rename `get_tokenizer_info` to `get_window_service_info` everywhere.

This is because if this were `get_tokenizer_info`, I would expect the parameter to be the tokenizer name, not the model name (which can be different). But we need this to be the model name for this PR to work.